mutation_partition.hh

```diff
@@ -878,6 +878,11 @@ public:
     friend std::ostream& operator<<(std::ostream& os, const printer& p);
 };
 
+struct mutation_application_stats {
+    uint64_t row_creations;
+    uint64_t row_overwrites;
+};
```

Comment:
You can use = 0; to initialize them here and not worry about forgetting to initialize them later.

Reply:
Done.